test/classes/Gis/GisVisualizationTest.php

```diff
@@ -120,19 +118,12 @@ public function testModifyQueryOld(): void
      */
     public function testModifyQuery(): void
     {
+        $this->dbi?->setVersion(['@@version' => '8.0.0']);
```

Comment:
```suggestion
        $this->dbi->setVersion(['@@version' => '8.0.0']);
```
I do not see why we should allow it to soft fail, the test needs it ;)

Reply:
This is also just a convenient way to ignore analyzer warnings. If you prefer, an `assert($this->dbi !== null)` would also work.